sql/step_04_all_following_steps.sql

```diff
@@ -104,3 +104,4 @@ source step_a107_add_permission_admin_access.sql;
 source step_a108_audit_dicom_type_and_dicom_data.sql;
 source step_a109_update_user_settings_table_constraints_and_add_usergroups_permissions.sql;
 source step_a110_update_operations_table_change_ope_for_to_enum.sql;
+source step_a112_add_missing_lock_columns.sql;
```

Comment:
Please rename it as probably been merged before OP-1310
```suggestion
source step_a111_add_missing_lock_columns.sql;
```